### PR TITLE
Updates to expand/collapse behavior

### PR DESCRIPTION
**Pull request by designedbyscience**
Makes panel expansion more sane, also addresses #2050

### CODE REVIEW

**Comment by designedbyscience** [comment]
Oh, and #2094

File: src/style/panel.less [hunk 1]
```diff
@@ -181,7 +181,7 @@
 
 .layers .section-container {
     padding: 0;
-    flex-shrink: 1;
+    flex-shrink: 1;
```
Comment (iwehrman) [inline src/style/panel.less]: bloop

**Comment by iwehrman** [comment]
So you're sticking with the whole "do nothing if you try to collapse the last open panel" thing? At least in the case of two-panels-per-column, this seems worse than the current behavior. But if you and @placegraphichere are on the same page, I'll go along!

**Comment by designedbyscience** [comment]
I said "more sane", not "completely sane"

**Comment by iwehrman** [comment]
More sane it is.

**Pull request merged**